documentation/README.md

```diff
@@ -11,6 +11,7 @@ When running a dotnet application, differences in diverse local and production e
 - Getting Started
     - [Running on a local machine](./localmachine.md)
     - [Running in Docker](./docker.md)
+    - [Running in Docker Compose](./docker-compose.md)
     - [Running in Kubernetes](./kubernetes.md)
     - Enabling SSL
```

Comment:
Out of scope of this PR, but should `Enabling SSL` be removed? It doesn't seem to be linked to anything

Reply:
https://github.com/dotnet/dotnet-monitor/pull/7277